### ISSUE TITLE
GroupNormalization relation to Instance and LayerNormalization

### ISSUE THREAD

**Issue by CNacherC-UV** [keras-team/tf-keras#195]
In the [GroupNormalization Layer documentation](https://keras.io/api/layers/normalization_layers/group_normalization/) , it says the following:

> "Relation to Layer Normalization: If the number of groups is set to 1, then this operation becomes nearly identical to Layer Normalization (see Layer Normalization docs for details).
> 
> Relation to Instance Normalization: If the number of groups is set to the input dimension (number of groups is equal to number of channels), then this operation becomes identical to Instance Normalization."

isn't Layer swapped for Instance ? Actually LayerNormalization is defined as that normalisation across al features (channels)

Best regards
